src/Command/CronTabCommand.php

```
@@ -0,0 +1,138 @@
+                if ($output->isVerbose()) {
+                    $io->text("Running: $cmd");
+                }
+
+                $cronInput = new ArrayInput([
+                    'command' => $cmd
+                ]);
+
+                // disable interactive behavior for the greet command
+                $cronInput->setInteractive(false);
+
+                $returnCode = $app->doRun($cronInput, $output);
```

PHPMD (2.15.0)

Avoid unused local variables such as '$returnCode'. (undefined)

(UnusedLocalVariable)